pkg/scheduler/factory/factory.go

```diff
@@ -387,19 +387,27 @@ func (c *configFactory) CreateFromConfig(policy schedulerapi.Policy) (*Config, e
 	var extenders []algorithm.SchedulerExtender
 	if len(policy.ExtenderConfigs) != 0 {
 		ignoredExtendedResources := sets.NewString()
+		var sortedExtenders []algorithm.SchedulerExtender
 		for ii := range policy.ExtenderConfigs {
 			klog.V(2).Infof("Creating extender with config %+v", policy.ExtenderConfigs[ii])
 			extender, err := core.NewHTTPExtender(&policy.ExtenderConfigs[ii])
 			if err != nil {
 				return nil, err
 			}
-			extenders = append(extenders, extender)
+			if extender.IsIgnorable() {
+				extenders = append(extenders, extender)
+			} else {
+				sortedExtenders = append(sortedExtenders, extender)
```

Comment:
It's actually not literally "sorted" - it's two types of extenders. I'd suggest to use `ignorableExtenders` to store ignorable entenders, and append nonIgnorableExtender to `extenders`.

And also L409~L410 should be changed to:

```go
extenders = append(extenders, ignorableExtenders...)
```

Reply:
Yeah - that would be cleaner.

Addressed